CI: type error

```log
Run pnpm run lint-check

> hono-tencent-cloudbase-cloud-function-adapter@1.0.0 lint-check /home/runner/work/hono-tencent-cloudbase-cloud-function-adapter/hono-tencent-cloudbase-cloud-function-adapter
> eslint . --no-fix


/home/runner/work/hono-tencent-cloudbase-cloud-function-adapter/hono-tencent-cloudbase-cloud-function-adapter/test-esm.mjs
Error:    5:1   error  'console' is not defined  no-undef
Error:    [6](https://github.com/zsnmwy/hono-tencent-cloudbase-cloud-function-adapter/actions/runs/15575351034/job/43859073564#step:6:7):1   error  'console' is not defined  no-undef
Error:    6:36  error  'process' is not defined  no-undef
Error:   15:18  error  'process' is not defined  no-undef
Error:   59:3   error  'console' is not defined  no-undef
Error:   60:3   error  'console' is not defined  no-undef
Error:   66:[7](https://github.com/zsnmwy/hono-tencent-cloudbase-cloud-function-adapter/actions/runs/15575351034/job/43859073564#step:6:8)   error  'console' is not defined  no-undef
Error:   68:7   error  'console' is not defined  no-undef
Error:   71:5   error  'console' is not defined  no-undef
Error:   74:3   error  'console' is not defined  no-undef
Error:   75:3   error  'process' is not defined  no-undef

✖ [11](https://github.com/zsnmwy/hono-tencent-cloudbase-cloud-function-adapter/actions/runs/15575351034/job/43859073564#step:6:12) problems (11 errors, 0 warnings)

 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

--- a/eslint.config.mjs
+++ b/eslint.config.mjs
@@ -8,6 +8,33 @@ export default tseslint.config(
   ...tseslint.configs.strict,
   ...tseslint.configs.stylistic,
   {
-    ignores: ["dist/*", "test-nodejs18.js", "example-commonjs.js", "scripts/*", "examples/*"],
+    ignores: ["dist/*", "scripts/*", "examples/*"],
+  },
+  {
+    // Node.js 环境配置 - 适用于测试文件和 CommonJS 文件
+    files: ["test-*.js", "test-*.mjs", "example-*.js", "*.config.js", "*.config.mjs"],
+    languageOptions: {
+      globals: {
+        console: "readonly",
+        process: "readonly",
+        Buffer: "readonly",
+        __dirname: "readonly",
+        __filename: "readonly",
+        global: "readonly",
+        module: "readonly",
+        require: "readonly",
+        exports: "readonly",
+        URL: "readonly",
+        Response: "readonly",
+        Request: "readonly",
+        fetch: "readonly",
+      },
+    },
+    rules: {
+      // 允许 CommonJS 文件使用 require
+      "@typescript-eslint/no-require-imports": "off",
+      // 允许未使用的变量（测试文件中常见）
+      "@typescript-eslint/no-unused-vars": "warn",
+    },
   }
 );

